tensorflow/compiler/mlir/tensorflow/transforms/convert_tf_control_flow_to_scf.cc

```diff
@@ -25,6 +25,22 @@ namespace TF {
 
 namespace {
 
+/// Move the ops of `source_block` into `destination_block`, keeping the later's
+/// block arguments' type as `block_arguments_type`.
+static void moveBlock(Block* source_block, Block* destination_block,
+                      TypeRange block_arguments_type,
+                      PatternRewriter& rewriter) {
+  // If `destination_block` isn't empty, erase its terminator to ensure that it
+  // never contains two terminator-like ops after merging.
+  if (!destination_block->empty())
+    rewriter.eraseOp(destination_block->getTerminator());
+
+  destination_block->addArguments(block_arguments_type);
+  rewriter.mergeBlocks(source_block, destination_block,
+                       destination_block->getArguments());
+  return;
```

Comment:
Can you remove this line? Our linter is a bit overly stricts and blocks on this.

Reply:
> Can you remove this line? Our linter is a bit overly stricts and blocks on this.

Remove the `return;`?

Reply:
Yes: this is a void function

Reply:
Done!